cfpb/teachers-digital-platform
# Reduces repository to just CRTool

Short description explaining the high-level reason for the pull request

## Additions

- Refactors structure to make crtool the only referenced application.

## Removals

- All of activity search and building blocks tools which are now located in cfgov-refresh

## Changes

- Leverage gulp for less compilation but npm scripts for building the minified js.

## Testing

- Pull into the develop-apps folder. 
- Run the setup.py script to install the javascript
- Open the CRTool and confirm css/js exists and is pulled in correctly.

## Review

- @Scotchester 
- @willbarton 
- @higs4281 
- @cwdavies 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots


## Notes

- Sorry for the mass amount of files changed.

## Todos

-

## Checklist
